src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_Flyout.cs

```diff
@@ -18,12 +18,6 @@
 using Windows.UI.Xaml.Media;
 using Windows.UI.Xaml.Shapes;
 
-#if HAS_UNO
-using Popup = Windows.UI.Xaml.Controls.Popup;
-#else
-using Popup = Windows.UI.Xaml.Controls.Primitives.Popup;
-#endif
-
```

Comment:
This was causing the build to fail. I don't know how it relates to my changes. The namespace has already changed earlier.